packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -76,6 +90,62 @@ class HorizontalOverflow extends Foundation<HorizontalOverflowProps,  React.AllH
         this.setState({
             itemsHeight
         });
+
+        if (canUseDOM() && this.horizontalOverflowItemsRef.current) {
+            this.horizontalOverflowItemsRef.current.addEventListener("scroll", this.requestFrame);
+        }
+    }
+
+    /**
+     * React life-cycle method
+     */
+    public componentWillUnmount(): void {
+        if (canUseDOM() && this.horizontalOverflowItemsRef.current) {
+            this.horizontalOverflowItemsRef.current.removeEventListener("scroll", this.requestFrame);
+        }
+    }
+
+    /**
+     * Callback on scroll change
+     */
+    public onScrollChange = (): IScrollObject => {
+        let scrollObject: IScrollObject;
+        const distanceRemaining: number =
+            this.horizontalOverflowItemsRef.current.scrollWidth - this.horizontalOverflowItemsRef.current.scrollLeft;
+
+        if (this.props.onScrollChange) {
+            this.props.onScrollChange();
```

Comment:
I think you want to pass the `scrollObject` as a parameter to `this.props.onScrollChange` - otherwise an implementer cant know anything about the nature of the scroll change.